wiki/subscriptions.md

```diff
@@ -34,7 +34,8 @@ the aggregated object will look like when writing the conditions. The
 requirements inside a subscription should reflect a wanted state of the 
 aggregation, for example: "_Hey Eiffel Intelligence, let me know when my 
 artifact has been published here!_" or "_I want to know when these two products 
-have reached confidence level X_".
+have reached confidence level X_". Requirements in subscriptions are written 
+using [JMESPath](https://jmespath.org/) language.
```

Comment:
I think it would be more clear if "Requirements in subscriptions are written 
using [JMESPath](https://jmespath.org/) language." was it´s own line before the text blob or after the text blob rather then a part of the text blob.

Reply:
done